hedera-node/pom.xml

```diff
@@ -306,6 +306,10 @@
       <groupId>net.i2p.crypto</groupId>
       <artifactId>eddsa</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+    </dependency>
```

Comment:
I am thinking module `hapi-fees` (probably other modules) also this log capture feature. Maybe this dependency can be moved to `hedera-services` level?

Reply:
Excellent point...I think to reuse the `LogCaptorExtension` between `hapi-fees` and `hedera-node`, we need to move it into a separate `test-libs` module...will do some more research and tackle this in a separate PR when we want to use it in `hapi-fees`. 👍